source/SpriteParameters.cpp

```diff
@@ -0,0 +1,157 @@
+/* SpriteParameters.cpp
+Copyright (c) 2024 by XessWaffle
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "SpriteParameters.h"
+
+#include "Sprite.h"
+using namespace std;
+
+
+SpriteParameters::SpriteParameters()
+{
+}
+
+
+
+SpriteParameters::SpriteParameters(const Sprite *sprite)
+{
+	AnimationParameters initDefault;
+	auto spriteDetails = SpriteParameters::SpriteDetails{sprite, initDefault, {}};
+	sprites.insert(pair<int, SpriteParameters::SpriteDetails>(0, spriteDetails));
+	exposed = initDefault;
```

Comment:
```suggestion
	auto spriteDetails = SpriteParameters::SpriteDetails{sprite, {}, {}};
	sprites.insert(pair<int, SpriteParameters::SpriteDetails>(0, spriteDetails));
```

Reply:
Good catch, I should look through the code and see if I can find any other places where I can use `{}`